docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -0,0 +1,176 @@
+[id="migrating-existing-daemons-to-target-nodes_{context}"]
+
+= Scenario 1: Migrating the existing daemons to the target nodes
+
+Assuming we have 3 {CephCluster} nodes or ComputeHCI, this scenario extends the
+“monitoring” labels to all the {CephCluster} (or ComputeHCI) nodes that are part
+of the cluster. This means that we keep the count: 3 placements for the target
+nodes.
+//kgilliga: Note to self: This intro will be rewritten for GA.
+
```

Comment:
No prerequisites needed, yes?

Reply:
I think the prerequisites from this module apply: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-monitoring-stack.adoc

This procedure will eventually be rewritten more thoroughly, and I can find out more then.